bindings/cs/MongoDB.Libmongocrypt/MongoDB.Libmongocrypt.targets

```diff
@@ -1,9 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <ItemGroup Condition="$([MSBuild]::IsOsPlatform('Windows'))">
-        <Content Include="$(MSBuildThisFileDirectory)../runtimes/win/native/mongocrypt.dll">
-            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-            <Link>mongocrypt.dll</Link>
-        </Content>
-    </ItemGroup>
+  <ItemGroup Condition="$([MSBuild]::IsOsPlatform('Windows'))">
+    <Content Include="$(MSBuildThisFileDirectory)../runtimes/win/native/mongocrypt.dll">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Link>mongocrypt.dll</Link>
+    </Content>
+  </ItemGroup>
+  <ItemGroup Condition="$([MSBuild]::IsOsPlatform('Linux'))">
+    <Content Include="$(MSBuildThisFileDirectory)../runtimes/linux/native/libmongocrypt.so">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Link>libmongocrypt.so</Link>
+    </Content>
+  </ItemGroup>
```

Comment:
IMHO these item groups shouldn't be conditional on the OS platform. We should copy `mongocrypt.dll`, `libmongocrypt.so`, and (eventually) `libmongocrypt.dylib` to the output directory so that the developer's compiled code can run on any supported platform. Right now if a developer includes our NuGet package and compiles on Windows, then only `mongocrypt.dll` will end up in the output folder. The app would fail if copied and run on Linux. The converse is true if compiled on Linux. `libmongocrypt.so` would be copied to the output folder and then you wouldn't be able to run on Windows.

Without the conditional statements, `mongocrypt.dll` and `libmongocrypt.so` (and eventually `libmongocrypt.dylib`) would be copied to the output folder and that compiled application could be run on any supported platform.

Reply:
>MongoDB.Libmongocrypt.targets 

this is used only for our local tests.

>so that the developer's compiled code can run on any supported platform

I'm not sure that this is the right statement, at least this is disputable. This is also can be unsafe and lead to packaging issues. For now, I just mimic the previously added logic. I would work on any significant changes here in the scope of different ticket

Reply:
I did some experimenting with `targets` and based on my tests, we don't even need it.

Reply:
we need it for .net framework